docs/generators.md

```diff
@@ -0,0 +1,53 @@
+# Generators
+
+**Generators** are classes that are used to render models for a given language.
+
+## Supported languages
+
+- [JavaScript](../src/generators/javascript/JavaScriptGenerator.ts),
+- [TypeScript](../src/generators/typescript/TypeScriptGenerator.ts),
+- [Java](../src/generators/java/JavaGenerator.ts).
+
+## Generator's options
+
+Each generator extends default options for the generator. It means that the generator can also have additional options.
+
+Options is passed as first argument to generator's constructor. Check the example:
+
+```ts
+const generator = new TypeScriptGenerator({ ...options });
+```
+
+Default options contains:
+
+| Option | Type | Description | Default value |
+|---|---|---|---|
+| `indentation` | Object | Options for indentation. | - |
+| `indentation.type` | String | Type of indentation. Its value can be either `SPACES` or `TABS`. | `SPACES` |
+| `indentation.size` | String | Size of indentation. | 2 |
+| `defaultPreset` | Object | Default preset for generator. For more information, read [customisation](./customisation.md) document. | _Implemented by generator_ |
+| `presets` | Array | Array contains **presets**. For more information, read [customisation](./customisation.md) document. | `[]` |
+
+Below is a list with additional options available for a given generator.
+
+### [TypeScript](../src/generators/typescript/TypeScriptGenerator.ts)
+
+| Option | Type | Description | Default value |
+|---|---|---|---|
+| `renderTypes` | Boolean | Renders or not signature for types. | `true` |
```

Comment:
`Renders or not signature for types` is hard to understand IMHO. Since it is a boolean type, its clear that it enables/disables something. What about something like: 

```suggestion
| `renderTypes` | Boolean | Render signature for types. | `true` |
```

Reply:
Yeah, it's better :)